patches/chrome-browser-global_keyboard_shortcuts_mac.mm.patch

```diff
@@ -2,7 +2,7 @@ diff --git a/chrome/browser/global_keyboard_shortcuts_mac.mm b/chrome/browser/gl
 index 733785418f6e0fd4090f2c5b82517d01982ab8a5..7b8a57bc712fe2ff3c82ad35f8261cc5451afea0 100644
 --- a/chrome/browser/global_keyboard_shortcuts_mac.mm
 +++ b/chrome/browser/global_keyboard_shortcuts_mac.mm
-@@ -166,6 +166,7 @@ const std::vector<KeyboardShortcutData>& GetShortcutsNotPresentInMainMenu() {
```

Comment:
?

Reply:
It was in a different commit which is because the previous Chrome rebase did not rebase these files properly.